### PR TITLE
Minor improvement to deepspeed utility comments

### PR DESCRIPTION
**Pull request by kaushikb11**
## What does this PR do?

Minor improvement to deepspeed utility comments

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Review by SeanNaren — APPROVED** [review]

File: pytorch_lightning/utilities/deepspeed.py [hunk 1]
```diff
@@ -12,23 +12,27 @@
...
+
+Example usage within the Lightning checkpoint directory where 'latest' is found:
+
+>>> from pytorch_lightning.utilities.deepspeed import convert_zero_checkpoint_to_fp32_state_dict
+
+# Lightning deepspeed has saved a directory instead of a file
+
+>>> save_path = "lightning_logs/version_0/checkpoints/epoch=0-step=0.ckpt/"
+>>> output_path = "lightning_model.pt"
+>>> convert_zero_checkpoint_to_fp32_state_dict(save_path, output_path)
```
Comment (ethanwharris) [inline pytorch_lightning/utilities/deepspeed.py:33]:
```suggestion
from pytorch_lightning.utilities.deepspeed import convert_zero_checkpoint_to_fp32_state_dict

Lightning deepspeed has saved a directory instead of a file

save_path = "lightning_logs/version_0/checkpoints/epoch=0-step=0.ckpt/"
output_path = "lightning_model.pt"
convert_zero_checkpoint_to_fp32_state_dict(save_path, output_path)
```

**Review by ethanwharris — COMMENTED** [review]
@kaushikb11 This seems to be showing up as a doctest, which fails presumably because deepspeed isn't installed. Maybe have it just as a comment but not a doctest.

File: pytorch_lightning/utilities/deepspeed.py (hunk 1 above)
Comment (kaushikb11) [reply pytorch_lightning/utilities/deepspeed.py:33]: @ethanwharris I could add `# doctest: +SKIP` to the code lines, right?

Comment (ethanwharris) [reply pytorch_lightning/utilities/deepspeed.py:33]: I think that should work too, worth a shot! :smiley:

**Review by ethanwharris — APPROVED** [review]

**Review by Borda — COMMENTED** [review]
If all is skip then don't use doctest

**Review by awaelchli — APPROVED** [review]

**Comment by awaelchli** [comment]
sorry, I didn't know it would merge. I thought @Borda reviewed but he only commented, which didn't block the merge. Sorry!
